### PR TITLE
Fixed issue #359

### PR DESCRIPTION
**Pull request by morrme**
Extracted construction into setUp() method per issue #359

### CODE REVIEW

File: exercises/diamond/src/test/java/DiamondPrinterTest.java [hunk 1]
```diff
@@ -39,7 +47,7 @@ public void testThreeByThreeDiamond() {
     @Ignore
     @Test
     public void testFiveByFiveDiamond() {
-        List<String> output = new DiamondPrinter().printToList('E');
+        List<String> output = diamondPrinter.printToList('E');
         assertThat(output, is(asList("    A    ",
                                      "   B B   ",
                                      "  C   C  ",
```
Comment (FridaTveit) [inline exercises/diamond/src/test/java/DiamondPrinterTest.java]: Looks like you missed one test :)

Comment (morrme) [inline exercises/diamond/src/test/java/DiamondPrinterTest.java]: Yikes! Should I fix and push again or did you edit it already?

File: exercises/diamond/src/test/java/DiamondPrinterTest.java [hunk 2]
```diff
@@ -9,17 +10,24 @@
 import static org.junit.Assert.assertThat;
 
 public final class DiamondPrinterTest {
-
+    private DiamondPrinter diamondPrinter;
+    
+    @Before
+
```
Comment (stkent) [inline exercises/diamond/src/test/java/DiamondPrinterTest.java]: This blank line between annotation and method signature is atypical; please remove it. Thanks! ❤️

**Comment by FridaTveit** [comment]
Thanks @morrme! :)

**Comment by morrme** [comment]
@FridaTveit @stkent  I appreciate the support! :)

**Pull request merged**